app/src/main/java/org/commcare/commcaresupportlibrary/identity/model/Identification.java

```diff
@@ -0,0 +1,120 @@
+package org.commcare.commcaresupportlibrary.identity.model;
+
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+
+@SuppressWarnings("unused")
+public class Identification implements Parcelable, Comparable<Identification> {
+
+    private String guid;
+    private int confidence;
+    private Tier tier;
+
+    /**
+     * Empty constructor
+     */
+    public Identification() {
+
+    }
+
+    /**
+     * This constructor creates a new identification
+     *
+     * @param guid       A string containing the guid
+     * @param confidence An int containing the (matching) confidence
+     */
+    public Identification(String guid, int confidence, Tier tier) {
```

Comment:
Confidence and Tier feel like they should either both be present everywhere or part of the same object. Different classifiers will have different scoring applications